Core/Object Arts/Dolphin/IDE/Base/SmalltalkSearchPresenter.cls

```diff
@@ -154,47 +151,47 @@ rangeToCompleteAt: anInteger
 				ifFalse: [anInteger to: view caretPosition - 1]]!
 
 searchItemsStartingWith: aString maxItems: anInteger
-	| result classes filter |
-	aString first isLowerCase
-		ifTrue: 
-			[| matchingSymbols |
-			matchingSymbols := self selectorsStartingWith: aString maxItems: anInteger.
-			^matchingSymbols].
-	filter := (aString includes: $>)
+	| result classes classFilter addBlock |
+	aString first isUppercase ifFalse: [^self selectorsStartingWith: aString maxItems: anInteger].
```

Comment:
Yes, precisely. (Other punctuation is also included, which allows searching for binary selectors.) I'll add a comment.